src/main/java/uk/gov/justice/hmpps/casenotes/controllers/CaseNoteController.java

```diff
@@ -102,4 +99,23 @@ public CaseNote amendCaseNote(
 
        return caseNoteService.getCaseNotes(filter, pageable);
     }
+
+    @GetMapping("/types")
+    @ApiOperation(value = "Retrieves a list of case note types", response = CaseNoteType.class)
```

Comment:
Is responseContainer=“list” needed here?

Reply:
👍

Reply:
Corrected